### PR TITLE
Fix tooltips on mobile devices

### LINKED ISSUE #6382: Missing tooltips on the Contao Backend on touch devices
It seems like there is no touch support for the explanation tooltips below each backend field on touch devices.

Tested on my iPhone 6s on safari.

### PR DESCRIPTION
**Pull request by fritzmg**
Fixes #6382 

Contao 4.13 will need a separate PR, as the tooltips are implemented via `Tips.Contao` in `mootao.js`.

### CODE REVIEW

**Comment by aschempp** [comment]
wasn't there a PR that changed the script to Stimulus? Maybe it would be easier to fix there and backport to 5.3?

File: core-bundle/assets/scripts/tips.js [hunk 1]
```diff
@@ -76,9 +78,20 @@
...
-                }, 100)
+                }, 'mouseleave' === e.type ? 100 : 0)
+            }
+        }
+
+        el.addEventListener('mouseleave', close);
+
+        // Close tooltip when touching anywhere else
+        document.addEventListener('touchend', (e) => {
+            if (e.originalTarget.contains(el)) {
```
Comment (ausi) [inline core-bundle/assets/scripts/tips.js]:
```suggestion
            if (e.target?.contains(el)) {
```
I think `originalTarget` is not supported everywhere and not needed here?

**Comment by zoglo** [comment]
> wasn't there a PR that changed the script to Stimulus? Maybe it would be easier to fix there and backport to 5.3?

But questionable if it's a real Stimulus Controller (: #7458

File: core-bundle/assets/scripts/tips.js (hunk 1 above)
Comment (fritzmg) [reply core-bundle/assets/scripts/tips.js]: Hm, I think it needs to be `el.contains(e.target)` anyway, doesn't it? I always get confused here 😁

Comment (ausi) [reply core-bundle/assets/scripts/tips.js]: Yes ☺️

```suggestion
            if (el.contains(e.target)) {
```

File: core-bundle/assets/scripts/tips.js [hunk 2]
```diff
@@ -76,9 +78,20 @@
                 timer = setTimeout(function() {
                     tip.style.display = 'none';
                     tip.style.willChange = 'auto';
-                }, 100)
+                }, 'mouseleave' === e.type ? 100 : 0)
+            }
+        }
```
Comment (leofeyer) [inline core-bundle/assets/scripts/tips.js]:
```suggestion
        };
```

**Comment by aschempp** [comment]
Is this something we really need to fix in 5.3? If I undestand correctly, the same issue needs to be fixed separately in 5.x because of the Stimulus controller?

**Comment by fritzmg** [comment]
> Is this something we really need to fix in 5.3?

Of course, why wouldn't we?

> If I undestand correctly, the same issue needs to be fixed separately in 5.x because of the Stimulus controller?

There is already a separate PR for that. See https://github.com/contao/contao/pull/7596

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @fritzmg.

**Pull request merged**